core/context_processors.py

```diff
@@ -8,6 +8,25 @@ def current_version(request):
     return {"current_version": Version.objects.most_recent()}
 
 
+def active_nav_item(request):
+    """Custom context processor that adds the active nav item to the context"""
+    default_nav_item = "home"
+    path_map = {
```

Comment:
I like that this is now a context processor. Do you think this map might be better if it's based on the view name?

`self.request.resolver_match.view_name`

Reply:
My thinking being that if the paths in the urls.py change we don't need to go hunting down everywhere they're used.

Reply:
I did think about using view names, but decided to go with paths, since that's what we were doing already in js and it works well. They seem more or less equivalent to me. Wouldn't your second comment apply if "the ~paths~ names in the urls.py change"? Currently name and path prefixes are mostly the same. I don't expect that will change.

If you feel strongly, I'm happy to change it.

Reply:
Would be great if we namespaced our urls by app, in which case we could probably use `resolver_match.namespace`.

Reply:
That's a fair point, I'd think those would come up in the search for the templates needing to change to match the path, and would be less likely to change than the paths in the first place. I'm not set on it or anything, just thought I'd bring it up and see what you thought. Entirely your call.

Reply:
Sorry, I'm not following - why would anything need to change in templates?

Reply:
In your scenario where the names change any templates referencing them would also need to be changed to match.

Reply:
I don't think any templates reference paths (nor would they reference names if we used them as keys here).